osc-ui/src/main/java/org/osc/core/broker/view/maintenance/PluginsLayout.java

```diff
@@ -100,15 +104,33 @@ public PluginsLayout(BundleContext ctx, ImportPluginServiceApi importPluginServi
         this.plugins.addContainerProperty(PROP_PLUGIN_INFO, String.class, null);
         this.plugins.addContainerProperty(PROP_PLUGIN_DELETE, Button.class, null);
 
+        this.plugins.setColumnWidth(PROP_PLUGIN_INFO, PLUGIN_INFO_COLUMN_WIDTH);
+
+        // Add a tooltip to the error column so the user is able to see the
+        // complete error message
+        this.plugins.setItemDescriptionGenerator(new ItemDescriptionGenerator() {
```

Comment:
you can use lamda expr.

I would recommend to enable the save action in java which automatically converts annoymous classes to lambdas.